rosdep/python.yaml

```
@@ -1941,6 +1941,13 @@ python-hypothesis:
   gentoo: [dev-python/hypothesis]
   ubuntu:
     '*': [python-hypothesis]
+python-ifaddr:
+  alpine: [py3-ifaddr]
+  arch: [python-ifaddr]
+  debian: [python3-ifaddr]
+  fedora: [python3-ifaddr]
+  ubuntu:
+    '*': [python3-ifaddr]
```

This should be simplified:

```suggestion
  ubuntu: [python3-ifaddr]
```